.flaskenv

```diff
@@ -4,6 +4,7 @@ DB_PASSWORD=postgres
 DB_PORT=5432
 FLASK_APP=conbench
 FLASK_ENV=development
+FLASK_DEBUG=true
```

Comment:
I'm 95% certain this `.flaskenv` is not used in production deployments, so it's all good to hard code debug to true here — but want to double check that you agree that's actually the case here